ios/EmailShare.m

```diff
@@ -10,18 +10,49 @@
 
 
 @implementation EmailShare
+
+- (NSString*)getExtensionFromBase64:(NSString*)base64String {
```

Comment:
I think we already have a function that handles `base64` , maybe we can give a look to avoid duplicated code.

Reply:
You are right, there's something similar, but it returns a hardcoded PNG if there are no matches, and it also performs a file save operation from an extra parameter containing data.  I didn't want to break that behaviour or add a bunch of different if conditions to handle each use.  If you can take a look and provide some recommendation it would be good.

Reply:
Moved this to an utils class.